src/Transitioner.js

```diff
@@ -57,7 +57,8 @@ const defaultRenderScreen = (
   </Animated.View>
 );
 
-const defaultRenderContainer = (children) => (
+const defaultRenderContainer = (transitionRouteKey, transitions, navigation,
```

Comment:
Maybe it'll be better to accept an object instead?

Reply:
Sounds like a good idea 👍